### ISSUE TITLE
feature request: add -forward-only option to not start engine if there is no engine already running

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#942]
Right now we can do things like `daemonded +devmap <parpax>` to forward the command to an existing engine already running. But if the engine is not already running, the engine will start.

This is sometime wanted, for example `daemon +connect <servername>` will either connect to the server, or start the game and connect to the server, which is the mechanism used by the `unv://` protocol, which is very cool.

This is sometime unwanted, like if you're a server owner and all you want to do is to forward a command to an existing server, and you only want to forward commands to existing servers. For example, let's say you have a systemd service managing the `daemonded` server, you may want to do `sudo -u unvanquished daemonded +say hello` and be sure the engine doesn't start an instance in your current tty if for some reason the engine is not running yet.

So I suggest to implement an option like `-forward-only` that would not start the engine if there is no existing instance to forward the command to.

**Comment by slipher** [comment]
Yeah I’ve thought about this too. We should have a -forward that errors if the engine isn’t already running and a -noforward that errors if the engine is already running.

**Comment by illwieckz** [comment]
Here is another day I would love to have this!

**Comment by illwieckz** [comment]
> We should have a -forward that errors if the engine isn’t already running and a -noforward that errors if the engine is already running.

I like that.
